### PR TITLE
Inject necessary pythonpaths so that new plugin paths are included

### PR DESCRIPTION
**Pull request by pawelcodilime**

### CODE REVIEW

File: psutil_agent/loop.py [hunk 1]
```diff
@@ -144,14 +140,18 @@ def collect_metrics(rabbit_config, psutil_config):
     scheduler.run()
 
 
-def main():
+if __name__ == '__main__':
```
Comment (dankilman) [inline psutil_agent/loop.py:150]: why did you remove the `main` function?

File: psutil_agent/loop.py [hunk 2]
```diff
@@ -144,14 +140,18 @@ def collect_metrics(rabbit_config, psutil_config):
...
     args = [json.loads(a.replace('\\"', '"')) for a in sys.argv[1:]]
-    rabbit_config, log_dir, psutil_config = args[0], args[1], args[2:]
+    rabbit_config, log_dir, paths, psutil_config = (args[0], args[1], args[2],
+                                                    args[3:])
 
-    logging.basicConfig(filename=os.path.join(log_dir, 'psutil.log'))
+    for p in paths:
```
Comment (dankilman) [inline psutil_agent/loop.py]:
``` python
if  p not in sys.path:
```

File: psutil_agent/loop.py (hunk 1 above)
Comment (pawelcodilime) [inline psutil_agent/loop.py:150]: Because imports made inside a function aren't available outside it.

**Comment by dankilman** [comment]
It seems flake8 is failing to to the double imports. We need to #noqa these lines.
also, did you test this again after the changes from the last comments I made?

**Comment by pawelcodilime** [comment]
OK, I used #noqa. I tested it by hand and ran system test.

**Comment by dankilman** [comment]
OK, looks good to merge. Please restore plugin.yaml and blueprint.yaml to their original state.
